### PR TITLE
refactor(pages): update onboarding roles

### PR DESCRIPTION
**Pull request by filipelima18**
## What kind of change does this PR introduce?

This PR updates the role options on the first post-signup question so that it matches the roles we have defined internally.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

I couldn't figure out a way to simulate a new user onboarding. Also, I'm not sure if this requires any back-end work so that the new roles get registered to our database. Ideally, that should be tested during the review.

Closes MKT-354

### CODE REVIEW

**Review by alexnm — APPROVED** [review]

File: packages/app/src/app/pages/SignIn/Onboarding.tsx [hunk 1]
```diff
@@ -8,11 +8,13 @@ import { StyledButton } from 'app/components/dashboard/Button';
...
 
 const ROLE_OPTIONS = [
-  { value: 'frontend', label: 'Frontend developer' },
-  { value: 'backend', label: 'Backend developer' },
-  { value: 'fullstack', label: 'Fullstack developer' },
+  { value: 'frontend', label: 'Front-End Developer' },
+  { value: 'backend', label: 'Back-End Developer' },
+  { value: 'fullstack', label: 'Full-Stack Developer' },
+  { value: 'designer', label: 'Lead Developer' },
+  { value: 'designer', label: 'Engineering Manager' },
```
Comment (alexnm) [inline packages/app/src/app/pages/SignIn/Onboarding.tsx:15]: I think there's a wrong value here for Lead Developer and Engineering Manager, they shouldn't have value `designer` right?

**Review by alexnm — DISMISSED** [review]

File: packages/app/src/app/pages/SignIn/Onboarding.tsx (hunk 1 above)
Comment (filipelima18) [reply packages/app/src/app/pages/SignIn/Onboarding.tsx]: True. Not sure how I missed that. Just committed an update. Can you have a look @alexnm ?

**Review by christianalfoni — APPROVED** [review]
This looks good to me! Though we should validate with BE if there are any expectations on values here 👍

**Review by filipelima18 — COMMENTED** [review]
Updated.

**Pull request merged**